docs/docs/tutorial/chapter1/file-structure.md

```diff
@@ -16,10 +16,18 @@ Don't worry about trying to memorize this directory structure right now, it's ju
 │   ├── db
 │   │   └── schema.prisma
 │   ├── dist
+│   ├── jest.config.js
+│   ├── jsconfig.json
+│   ├── package.json
+│   ├── server.config.js
 │   ├── src
 │   │   ├── directives
 │   │   │   ├── requireAuth
+│   │   │   │   ├── requireAuth.js
```

Comment:
I included every file. Happy to revert these changes if they were left out on purpose previously.

Reply:
@Tobbe I think it's ok to include requireAuth, but config files were excluded:

https://github.com/redwoodjs/redwood/blob/fa5baa64eb4bedfa090c0019cd095aec6098aef3/docs/docs/tutorial/chapter1/file-structure.md?plain=1#L3

I removed them in https://github.com/redwoodjs/redwood/pull/8959/commits/20e2834947bfc9b28f0469ff7453c0d02c4c4307

Reply:
@Tobbe I think I'm just gonna get Rob's input on this cause there's more nuanced changes here (in the original) than I thought

Reply:
Yeah I excluded config files because there were so many, and weren't relevant to the app we're building. I'd consider `entry.client.jsx` a config file and wouldn't include it in this list, personally.

Reply:
Is the `/dist` directory present in a brand new app created with CRA? It didn't used to be I don't think, and so wasn't included in this list. 

And I think I just stuck with just the directories for directives since we don't go into the details of them during the tutorial—no reason to have to pack even *more* filenames into your brain that you don't absolutely need to understand at this point.

Reply:
Removed `/dist` for both web and api. Also removed the directive files.
But did decide to keep `entry.client.jsx`. It's kind of like an index file, and we do have `index.html` included. And personally I think it's easier to understand what's going on when you know where execution starts, which is in the entry file. So I decided to keep it.